### DISCUSSION TITLE
Subscriptions Counter Bug

### DISCUSSION THREAD

**Discussion by Pranav-yadav** [community/community#11852] [Other Feature Feedback, Questions, & Ideas]
## BUG:
_Just heads up, tried refreshing/reloading the page several times.._
1. The Subscriptions counter under [Subscriptions](https://github.com/notifications/subscriptions) tab (Notifications / Subscriptions) is showing incorrect count. As shown in below ss I've subscribed to only `1` discussions, tho it's showing `9`

## <details><summary>Screenshots</summary><img src="https://user-images.githubusercontent.com/55224033/154950201-83bffec6-8582-47de-86f6-36411e1a6063.png" alt="Subscriptions Counter Bug" /></details>

Filter (None using, defaults) tho,
- Reason _set to All_
- Repository _set to All_
- Sort _Most Recently ..._

2. Also, if only _Repository_ filter is applied it shows **No results matched** and the couter value is always `> 0`

Am I interpreting it wrong ? or it's a bug

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Comment by yan12125** [comment]
For me, the counter is higher than the number of my subscribed issues/PRs as subscribed commits are not displayed. I need to click on Repository to see what repositories contain commits that I subscribed, and find the actual commit from notification mails for those repos.
